Formula/automysqlbackup.rb

```diff
@@ -56,4 +58,7 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
```

Comment:
That'd be ideal, yeh. You could manually start a database in the test. Alternatively, having it do an `assert_match` on a  `--version` would be great at a minimum. Thanks!